lte/gateway/python/magma/mobilityd/tests/rpc_servicer_tests.py

```diff
@@ -329,6 +330,30 @@ def test_get_ip_for_unknown_subscriber(self):
         self.assertEqual(err.exception.code(),
                          grpc.StatusCode.NOT_FOUND)
 
+    def test_get_gw_info(self):
+        gw_info = self._stub.GetGatewayInfo(Void())
+        gw_ip_get = str(ipaddress.ip_address(gw_info.ip.address))
+
+        def_gw_cmd = "ip route show |grep default| awk '{print $3}'"
+        p = subprocess.Popen([def_gw_cmd], stdout=subprocess.PIPE, shell=True)
```

Comment:
I'd recommend always setting shell to False, even though it's a unit test, if we can avoid (shell=True) the better. You can use `shlex.split()` to get a list of params and use that to accomplish the same.